File: The Core/makeArrayConsecutive2.cs
int makeArrayConsecutive2(int[] statues) {
    int noStatuesNeeded = 0;
    Array.Sort(statues);

    for(int i = 0; i < statues.Length - 1; i++){
        int num = statues[i + 1] - statues[i];
        noStatuesNeeded += num - 1;
    }
    
    return noStatuesNeeded;
}

File: The Core/List Forest Edge/42. Make Array Consecutive 2/makeArrayConsecutive2.py
def makeArrayConsecutive2(statues):
    noStatuesNeeded = 0
    statues.sort()
    
    for x in range(0, len(statues) - 1):
        num = statues[x + 1] - statues[x]
        noStatuesNeeded += num - 1
        
    return noStatuesNeeded

#
#def makeArrayConsecutive2(statues):
#	return sum([1 for i in range(min(statues), max(statues)) if i not in statues])
